packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1012,6 +1015,7 @@ class _Editable extends LeafRenderObjectWidget {
       obscureText: obscureText,
       cursorWidth: cursorWidth,
       cursorRadius: cursorRadius,
+      textSelectionDelegate: textSelectionDelegate,
```

Comment:
you should also add this to updateRenderObject, for completeness (it can't currently change, but if we ever change the way this is built, we'd have a weird latent bug that's hard to track down if we don't do it correctly now)

Reply:
Alright did that!